src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/ProjectTelemetry/HashedString.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.LanguageServer.HostWorkspace.ProjectTelemetry;
+
+internal record HashedString(string HashedValue);
```

Comment:
Looks like it needs a custom serializer from a cursory investigation.  I think the point is so that it cannot get confused for a regular string once its hashed.  For now will leave this as-is

Reply:
I changed my mind, removed.